pkgs/development/python-modules/jenkins-job-builder/default.nix

```
@@ -17,6 +17,9 @@ buildPythonPackage rec {
   };
 
   postPatch = ''
+    # relax version constraint
```

Would help to add the upstream issue. The issue tracker of OpenStack's subprojects is "multiple clicks away".

```suggestion
    # relax version constraint, https://storyboard.openstack.org/#!/story/2009723
```